gui/doc/viselements.json

```diff
@@ -0,0 +1,1252 @@
+{
+  "controls": [
+    [
+      "text",
+      {
+        "default": "value",
+        "inherits": [
+          "on_change",
+          "shared"
+        ],
+        "properties": [
```

Comment:
properties: { name: {... details}} would insure unicity

Reply:
I see : order or unicity, you chose

Reply:
Correct. Another requirement is that it should be quite easy to update...